src/helper/build.ts

```diff
@@ -57,6 +59,9 @@ interface DevReactBuild {
 
 export function devBuild({ buildFolder, imageCdnUrl, isProd } : DevBuild) {
     const VUE_CLI_PATH = path.join('.', 'node_modules', '@vue', 'cli-service', 'bin', 'vue-cli-service.js');
+    const THEME_ENTRY_FILE = path.join('theme', 'index.js');

```

Comment:
Check is this needed ? @manishprajapati52

Reply:
Removed